tests/src/python/test_provider_postgres.py

```diff
@@ -2547,6 +2549,8 @@ def testFeatureCountEstimatedOnView(self):
             'test', 'postgres')
         self.assertTrue(vl.isValid())
         self.assertTrue(vl.featureCount() > 0)
+        vl.setSubsetString('"pk" = 3')
+        self.assertEqual(vl.featureCount(), 1)
```

Comment:
This is an estimation so we cannot be sure that the count would be one. Maybe we should just check that it is greater than 0